### ISSUE TITLE
Validators import not mentioned in example

### ISSUE THREAD

**Issue by digicos** [vue-generators/vue-form-generator-docs#6]
it is not clear from the documentation how to import the validator used in the example on this page
[Usage Example](https://icebob.gitbooks.io/vueformgenerator/content/usage.html)

checked the above issue at this [documentation  page](https://vue-generators.gitbook.io/vue-generators/usage) as well

**Comment by zoul0813** [comment]
Can you be more specific? What validator?

The new feature that allows you to import validators with `Vue.use(VueFormGenerator, options)` can be found under the [Validation](https://vue-generators.gitbook.io/vue-generators/validation) section of the documentation.

Please refer to the updated documented located at [https://vue-generators.gitbook.io/vue-generators/](https://vue-generators.gitbook.io/vue-generators/)

**Comment by digicos** [comment]
Thanks!. I referred to an outdated documentation. 
I took an example from the old documentation and that did not have hints to import the built in validators. (New documentation and JsFiddle has a good example though)
for e.g
The `validator: validators.string` mentioned on this [page](https://vue-generators.gitbook.io/vue-generators/validation) does not provide a hint about 'validators', how to import them?. I feel a small section mentioning it should be useful.

Thanks, for sharing vue-form-generator, a wonderful piece of work by you guys.
